### DISCUSSION TITLE
Find path based on fully qualified namespace (using PSR-4 from composer.json)

### DISCUSSION THREAD

**Discussion by ruudk** [rectorphp/rector#6953] [Q&A]
I want to move a class from `A\A\A` to `A\B\B`. 

When using `$this->removedAndAddedFilesCollector->addMovedFile()` I need to provide the new file path. 

Is there a way to convert `A\B\B` to the correct file path according to PSR-4 (composer.json)?

**Comment by TomasVotruba** [comment]
I'm not aware of such service. Maybe check the PSR4 rules for more clues.
